web/src/lib/components/shared-components/side-bar/recent-albums.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { getAssetThumbnailUrl } from '$lib/utils';
+  import { getAllAlbums, type AlbumResponseDto } from '@immich/sdk';
+  import { handleError } from '$lib/utils/handle-error';
+  import { t } from 'svelte-i18n';
+
+  let albums: AlbumResponseDto[] = $state([]);
+
+  onMount(async () => {
+    try {
+      const allAlbums = await getAllAlbums({});
+      albums = allAlbums.slice(0, 3);
```

Comment:
The albums are currently sorted by ascending creation date. How about sorting them by most recently updated instead?